### DISCUSSION TITLE
How to use a qualified operator?

### DISCUSSION THREAD

**Discussion by AZMCode** [koka-lang/koka#480] [Q&A] [answered]
I've been trying out the relatively (I took a while to get here) new implicit arguments, and they've been great so far, but I've fallen into a syntax error problem trying to use them.

Suppose I am defining a function, with an implicit operator argument as follows:
```koka
pub fun myfunc(a: int64, b: int64,?qualified/(+): (int64, int64) -> total int64): int64
    a ?qualified/(+) b
```
This code yields a
```
invalid syntax
unexpected identifier (operator) "@implicit/qualified/(+)"
expecting ".", "(", "[", "{", "fn", operator, ":=", ";", "return" or "}"
```
error. 
Is there a way to use the operator without removing the qualification?

Edit: On second look it appears the argument being implicit has nothing to do with this

**Comment by AZMCode** [comment, marked as answer]
I have found that using
```
(?qualified/(+))(a,b)
```
Works, but it is quite verbose

**Reply by chtenb** [reply]
I don't think you're supposed to write the question mark at the call site. This also compiles:

```koka
pub fun myfunc(a: int64, b: int64,?qualified/(+): (int64, int64) -> total int64): int64
  qualified/(+)(a, b)
```
